common-test/src/main/java/io/customer/commontest/extensions/IntExtensions.kt

```diff
@@ -0,0 +1,9 @@
+package io.customer.commontest.extensions
+
+import java.util.Random
+
+fun Int.Companion.random(min: Int, max: Int): Int {
```

Comment:
The extension was available in production code of `tracking` module. Since we only need this for tests, I moved them up here to `commontest` module.

Since `tracking` module uses in production code as well, the imports are not changed for `tracking` and only updated for rest of the modules.